src/mixins/helpers.js

```diff
@@ -15,9 +15,9 @@ var helpers = {
     var slideWidth;
 
     if (!props.vertical) {
-      slideWidth = trackWidth/props.slidesToShow;
+      slideWidth = this.getWidth(ReactDOM.findDOMNode(this))/props.slidesToShow;
```

Comment:
@akiran I changed this line when I wrote the vertical feature to be ReactDOM.findDOMNode(this.track) - which is stored in the trackWidth variable in the diff on the left - because I had the same thought. That's what seems to have causes this bug in the first place!